<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAppraiserDialog</class>
 <widget class="QDialog" name="AddAppraiserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>录入鉴定人信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="LBAppraiser">
       <property name="text">
        <string>鉴定人：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEAppraiser"/>
     </item>
     <item>
      <widget class="QLabel" name="LBAppraiserID">
       <property name="text">
        <string>《司法鉴定人执业证》证号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEAppraiserID"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="LBAppraiserPhoto">
         <property name="text">
          <string>鉴定人执业照</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnAppraiserPhoto">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="LBPhoto">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="BtnOK">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
